std/process.d

```diff
@@ -98,6 +98,9 @@ version (Windows)
     import std.windows.syserror;
 }
 
+import core.exception : OutOfMemoryError;
+
+import std.exception : enforce;
```

Comment:
Sorry, why were the local imports moved to a global import? I think that's opposite to current Phobos policy.

Reply:
I didn't care about, thank you for remind! Fixed at https://github.com/dlang/phobos/pull/7150